Library/PKPaymentAuthorizationViewController+Helpers.swift

```diff
@@ -0,0 +1,72 @@
+import KsApi
+import PassKit
+
+extension PKPaymentAuthorizationViewController {
+  public static var merchantIdentifier: String {
+    return Secrets.ApplePay.merchantIdentifier
+  }
+
+  public static var allSupportedNetworks: [PKPaymentNetwork] = [
+    .amex,
+    .masterCard,
+    .visa,
+    .discover,
+    .JCB,
+    .chinaUnionPay
+  ]
+
+  public static func applePayCapable() -> Bool {
+    return PKPaymentAuthorizationViewController.canMakePayments(usingNetworks: self.allSupportedNetworks)
+  }
+
+  public static func applePayDevice() -> Bool {
+    return PKPaymentAuthorizationViewController.canMakePayments()
+  }
+
+  public static func applePayCapable(for project: Project) -> Bool {
+    return PKPaymentAuthorizationViewController.canMakePayments(
+      usingNetworks: PKPaymentAuthorizationViewController.supportedNetworks(for: project)
+    )
+  }
+
+  public static func supportedNetworks(for project: Project) -> [PKPaymentNetwork] {
+    guard let availableCardTypes = project.availableCardTypes else {
```

Comment:
`availableCardTypes` should never be `nil` in this context, but decided to add a fallback just in case 🤷‍♀